fuel-vm/src/interpreter/flow/jump_tests.rs

```diff
@@ -8,7 +8,7 @@ use crate::error::PanicOrBug;
 #[test_case(0, 80, 10 => Ok(40); "jump backwards")]
 #[test_case(0, 40, VM_MAX_RAM => Err(PanicOrBug::Panic(PanicReason::MemoryOverflow)); "jump too far forward")]
 fn test_absolute_jump(is: Word, mut pc: Word, j: Word) -> SimpleResult<Word> {
-    JumpArgs::new(JumpMode::Absolute)
+    JumpArgs::new(JumpMode::RelativeIS)
```

Comment:
Should we add test cases in this file for `JumpMode::Assign`?

Reply:
Good point. Added in bb9d2161d.

Reply:
thanks!